fem/ceed/util.hpp

```diff
@@ -64,6 +74,14 @@ void InitBasisAndRestriction(const mfem::FiniteElementSpace &fes,
                              Ceed ceed, CeedBasis *basis,
                              CeedElemRestriction *restr);
 
+void InitTensorRestriction(const FiniteElementSpace &fes,
+                           Ceed ceed, CeedElemRestriction *restr);
+
+int CeedOperatorGetActiveField(CeedOperator oper, CeedOperatorField *field);
+
+int CeedOperatorGetActiveElemRestriction(CeedOperator oper,
```

Comment:
Rename to `GetOperatorActiveElemRestriction`?

Reply:
ditto.